DataFormats/L1TParticleFlow/interface/jets.h

```diff
@@ -83,7 +92,8 @@ namespace l1ct {
       j.v3.pt = CTtoGT_pt(hwPt);
       j.v3.phi = CTtoGT_phi(hwPhi);
       j.v3.eta = CTtoGT_eta(hwEta);
-      j.z0 = 0;
+      j.z0 = z0;
```

Comment:
```suggestion
      j.z0(l1ct::z0_t::width - 1, 0) = hwZ0(l1ct::z0_t::width - 1, 0);
```
(the LSB is the same, but l1ct objects use ap_uint and not ap_ufixed so you need to assign the bits)